compiler/main/configured_prefix.h

```diff
@@ -21,8 +21,10 @@
 #ifndef _configured_prefix_H_
 #define _configured_prefix_H_
 
-static const char* CONFIGURED_PREFIX =
-#include "CONFIGURED_PREFIX"
-;
+// TODO: THIS IS A TEMPORARY WORKAROUND for CMake SHould just get this
+// from CMAKE_INSTALL_PREFIX
+static const char* CONFIGURED_PREFIX = "";
+// #include "CONFIGURED_PREFIX"
```

Comment:
We should un-comment this for `main`, otherwise `make install` does not work, even if you comment it out again for your own testing. Of course, eventually, we need the PR that sorts it out correctly.